Makefile

```
@@ -18,6 +23,31 @@ endef
 help:
 	@python3 scripts/make_help.py Makefile
 
+
+# -------- Set up --------
+
+.PHONY: setup
+
+## Install all required development tools
+setup:
```

checkmake (0.2.2)

[warning] Target body for "setup" exceeds allowed length of 5 (17).

(maxbodylength)